httpbinding/encode.go

```diff
@@ -4,9 +4,12 @@ import (
 	"fmt"
 	"net/http"
 	"net/url"
+	"strconv"
 	"strings"
 )
 
+const contentLengthHeader = "Content-Length"
+
 // An Encoder provides encoding of REST URI path, query, and header components
 // of an HTTP request. Can also encode a stream as the payload.
 //
```

Comment:
May be call out the behavior for content-length header here. Just for visibility.

Reply:
Documented it on `Encode` call specifically.